src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -427,6 +427,8 @@ well as menu structures (for main menu and popup menus).
          <shortcut refid="jumpTo" value="Shift+Alt+J" if="!org.rstudio.core.client.BrowseCap.isMacintosh()"/>
          <shortcut refid="jumpTo" value="Cmd+Shift+Alt+J" if="org.rstudio.core.client.BrowseCap.isMacintosh()"/>
          <shortcut refid="jumpToMatching" value="Ctrl+P"/>
+         <shortcut refid="selectToMatching" value="Ctrl+Shift+P"/>
+         <shortcut refid="expandToMatching" value="Ctrl+Shift+Alt+P"/>
```

Comment:
On the Mac Ctrl+Shift+Alt+P inserts a special character for me so I don't think we can use it.